tests/cross/src/test/scala/tests/pc/SignatureHelpPatternSuite.scala

```diff
@@ -54,13 +56,19 @@ class SignatureHelpPatternSuite extends BaseSignatureHelpSuite {
       |case class Two[T](a: T, b: T)
       |object Main {
       |  (null: Any) match {
-      |    case Two(@@) =>
+      |    case Two("", @@) =>
       |  }
       |}
       |""".stripMargin,
     """|unapply(a: T, b: T): Two[T]
-       |        ^^^^
-       |""".stripMargin
+       |              ^^^^
+       |""".stripMargin,
+    compat = Map(
+      "3" ->
+        """|Two[T](a: T, b: T)
```

Comment:
I reworked the unapply signature for case classes to show the actual case classes. It's a bit different than in case of Scala 2, but I think this is a bit better since `def unapply(a: T, b: T): Two[T]` does not exist. The alternative would be to show the parameters alone.